go.mod

```diff
@@ -3,72 +3,71 @@ module github.com/aws-observability/aws-otel-collector
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsprometheusremotewriteexporter v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxrayexporter v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/dynatraceexporter v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/logzioexporter v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sapmexporter v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/signalfxexporter v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/awsproxy v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/healthcheckextension v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/ecsobserver v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/pprofextension v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/attributesprocessor v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/cumulativetodeltaprocessor v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatorateprocessor v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricsgenerationprocessor v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricstransformprocessor v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/probabilisticsamplerprocessor v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourceprocessor v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/spanprocessor v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsecscontainermetricsreceiver v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsxrayreceiver v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/statsdreceiver v0.43.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.43.0
 	github.com/opencontainers/runc v1.0.3
```

Comment:
Can you bump up runc while you're at it?

Reply:
Bumped to 1.1